source/gloperate/include/gloperate/painter/AbstractOutputCapability.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
```

Comment:
instead, the ```Data``` class should be included here (and I know, that this don't make any change to the compiler, but the separation of declaration and definition is much more clear).